pandas/core/indexes/timedeltas.py

```diff
@@ -112,6 +112,13 @@ def _engine_type(self) -> type[libindex.TimedeltaEngine]:
     # Use base class method instead of DatetimeTimedeltaMixin._get_string_slice
     _get_string_slice = Index._get_string_slice
 
+    # error: Return type "None" of "_resolution_obj" incompatible with return type
+    # "Resolution" in supertype "DatetimeIndexOpsMixin"
+    @property
+    def _resolution_obj(self) -> None:  # type: ignore[override]
+        # not used but need to implement it because it is an abstract method
+        return None
```

Comment:
why do we need to disable this?  it seems to work fine ATM

Reply:
Not sure why I disabled it - enabled it again (compared to the other classes it seems to be `Resolution | None`)